components/colors.ts

```diff
@@ -8,6 +8,7 @@ export const GREY_6 = "#52627b";
 export const GREY_7 = "#3e4e67";
 export const GREY_8 = "#354153";
 export const GREY_9 = "#2e3745";
+export const GREY_10 = "#cfcff5";
```

Comment:
I know it's following the designs, but we might be approaching the point where there are too many grays. That's a conversation to be had with the entire group though, so I'm just rolling with the colors the designs used for now